### DISCUSSION TITLE
Add distinct Maven goals

### DISCUSSION THREAD

**Discussion by helpermethod** [georgberky/dependency-update-maven-plugin#64] [Ideas]
* update
* updateDependencies
* updateDependencyManagement
* updateParent

`perform(<list of updates>)`

**Comment by bmarwell** [comment]
update plugins eventually? Or out-of-scope?

**Comment by helpermethod** [comment]
Should be pretty easy to do, very similar to updating regular dependencies.

The trickiest part is probably to get the selector right 😂.

**Comment by ascheman** [comment]
I would appreciate a `help` goal as with most other elaborated Maven plugins.
